cost-analyzer/charts/prometheus/README.md

```diff
@@ -63,8 +63,8 @@ Assuming you have an existing release of the prometheus chart, named `prometheus
 	  enabled: false
 	alertmanagerFiles:
 	  alertmanager.yml: ""
-	kubeStateMetrics:
-	  enabled: false
+	kube-state-metrics:
+	  disabled: true
```

Comment:
not ready for this...

Reply:
I must be missing something.  Are the two not equivalent?

Reply:
My apologies, missed that we changed for backwards compat. I'm good if you guys are.